protocol/src/types/block.rs

```diff
@@ -53,17 +54,33 @@ pub struct Proposal {
     pub prev_state_root:          MerkleRoot,
     pub transactions_root:        MerkleRoot,
     pub signed_txs_hash:          Hash,
-    #[cfg_attr(feature = "hex-serialize", serde(serialize_with = "serialize_uint"))]
+    #[cfg_attr(feature = "hex-serialize", serde(serialize_with = "serialize_uint",))]
+    #[cfg_attr(
```

Comment:
I do not understand the necessity of derive custom `Deserialize` function.

Reply:
forcerelay will call methods like get_block_by_id, the datas gotten should be deserialized to construct correct data structures